CMakeLists.txt

```diff
@@ -173,6 +173,12 @@ if (ENABLE_TENSORFLOW)
     createExtension(TENSORFLOW-EXTENSIONS "TENSORFLOW EXTENSIONS" "This enables TensorFlow support" "extensions/tensorflow" "${TEST_DIR}/tensorflow-tests")
 endif()
 
+## Bustache/template extensions
+option(ENABLE_BUSTACHE "Enables Bustache (ApplyTemplate) support." OFF)
```

Comment:
These seem to conflict.

Reply:
What with?

Reply:
Your conditional is DISABLE_BUSTACHE. Your option is ENABLE_BUSTACHE. It seems to only be enabled if you DISABLE_BUSTACHE ?

Reply:
Ah, my bad. Fixing that uncovered a missing include, but both issues are resolved in the latest push.

Reply:
Thanks. I'll merge shortly